azure/packages/test/end-to-end-tests/package.json

```diff
@@ -28,7 +28,7 @@
 		"test:coverage": "c8 npm test",
 		"test:realsvc": "npm run test:realsvc:tinylicious",
 		"test:realsvc:azure": "cross-env FLUID_CLIENT=azure npm run test:realsvc:azure:run",
-		"test:realsvc:azure:run": "mocha --recursive \"dist/test/**/*.spec.js\" --exit --timeout 20000 --config src/test/.mocharc.js",
+		"test:realsvc:azure:run": "mocha --recursive \"dist/test/**/*.spec.*js\" --exit --timeout 20000 --config src/test/.mocharc.js",
```

Comment:
Probably not a big deal, but `?` seems more correct if it works.

Reply:
I tried that first and didn't seem to work, but just checked and globbing should support it. I'll give it another shot just in case.

Reply:
I agree except that we shouldn't preclude `.js` files. `?` matches a single character. The more precise pattern is`{m,c,}` or use `{mjs,cjs,js}` for clarity.

Reply:
Ok it does work, but I'm still a bit hesitant. Are all packages dual-emitting now, so no plain `.js` files in any output anymore? `.*js` would support both if we still need it through a transition period. I'm partial to merging like this first.

Reply:
I agree to merge as-is. It is a bit loose, but `.spec.foo.js` shouldn't be a problem. The simplicity and flexibility is good.